src/validation.cpp

```diff
@@ -4192,42 +4192,63 @@ bool LoadMempool(void)
         if (version != MEMPOOL_DUMP_VERSION) {
```

Comment:
The implications seem to be minor, but this means all bitcoin nodes prior to this PR being merged will drop all their mempools on startup. Is that okay? Would it be possible / worth it to load version=1 mempools too? Above comment by @sipa seems to indicate this was never used, in which case I think we should simplay say `MEMPOOL_DUMP_VERSION  = 1` above.